hls-test-utils/hls-test-utils.cabal

```diff
@@ -56,9 +53,8 @@ library
     , tasty-rerun
     , temporary
     , text
-    , unordered-containers
     , row-types
-  ghc-options:      -Wall
+  ghc-options:      -Wall -Wunused-packages
```

Comment:
Should this be in a common section?

Reply:
This particular package only has one component (library) and no tests etc. So I felt it's overkill to use `common warnings` because it would only be imported in one place.